pkg/sql/information_schema.go

```diff
@@ -44,6 +44,7 @@ var informationSchema = virtualSchema{
 		informationSchemaTablesTable,
```

Comment:
Looks like this got out of alphabetical order. Mind fixing it and maybe adding a small comment?